middleware/limiter/limiter_fixed.go

```diff
@@ -69,12 +72,6 @@ func (FixedWindow) New(cfg Config) fiber.Handler {
 		// Set how many hits we have left
 		remaining := cfg.Max - e.currHits
 
-		// Update storage
-		manager.set(key, e, cfg.Expiration)
```

Comment:
think moving the set function must also be done in the part where the limit was reached